tests/eko/scale_variations/test_expanded.py

```diff
@@ -43,13 +43,12 @@ def test_singlet_sv_dispacher():
 def test_scale_variation_a_vs_b():
     r"""
     Test ``ModSV=exponentiated`` kernel vs ``ModSV=expanded``.
-    We test that the quantity :math:`(ker_A - ker_B)/ker_{unv}` depends
-    only on the accuracy in the :math:`\alpha_s` expansion
-    and not in the size of the `fact_to_ren` itself.
-    However in our implementation the exponentiated mode depends on
-    the actual value of a0 and a1, since the evolution integral in :math:`\alpha_s`
-    is evaluated. Thus this test ratio :math:`(ker_A - ker_B)/ker_{unv}`
-    still contains a dependency on `fact_to_ren`.
+    We test that the quantity :math:`(ker_A - ker_B)/ker_{unv}`
+
+    Note this is NOT the real difference between scheme expanded
+    and exponentiated since here we don't take into account the
+    shifts in path length and :math:`\alpha_s` values.
```

Comment:
Why don't we also shift the alpha values, as we should? we know how to do it (it's written in the notebook)

Reply:
This is just laziness. As mentioned above the tests will not test the real HO difference A-B, since we have verified that is too complex.